### PR TITLE
Customer note model

### PR DESCRIPTION
**Pull request by igorsergiichuk**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #

## The problem

CustomerNote model was absent

## How I resolved it

Added CustomerNote model

## How to verify it

Test it!

Notify the following people: @darrynten

### CODE REVIEW

File: src/Models/CustomerNote.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    /**
+     * Features supported by the endpoint
+     *
+     * @var array $features
+     */
+    protected $features = [
+        'all' => true,
+        'get' => true,
+        'save' => true,
+        'delete' => true
```
Comment (darrynten) [inline src/Models/CustomerNote.php]: There's a missing trailing comma here :)

**Review by darrynten — CHANGES_REQUESTED** [review]
Just a missing comma, looking good

File: tests/mocks/CustomerNote/GET_CustomerNote_Get.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "TotalResults": 3,
+  "TotalResults": 1,
```
Comment (darrynten) [inline tests/mocks/CustomerNote/GET_CustomerNote_Get.json]: This won't be valid you'll need to match the numbers you can't return more than you have

**Review by darrynten — CHANGES_REQUESTED** [review]

File: tests/mocks/CustomerNote/GET_CustomerNote_Get.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "TotalResults": 1,
```
Comment (darrynten) [inline tests/mocks/CustomerNote/GET_CustomerNote_Get.json]: This number is incorrect

**Review by darrynten — CHANGES_REQUESTED** [review]

File: tests/mocks/CustomerNote/GET_CustomerNote_Get.json (hunk 3 above)
Comment (igorsergiichuk) [reply tests/mocks/CustomerNote/GET_CustomerNote_Get.json]: done

**Review by darrynten — APPROVED** [review]
🚀

**Pull request merged**